sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,11 +1,28 @@
 # Release History
 
-## 12.0.0-beta.8 (Unreleased)
+## 12.0.0 (2021-06-11)
+
+### Bug fixes
+- Fixed issue where clients builders would not throw when calling `buildClient()` or `buildAsyncClient()` if not `endpoint` had been set.
+- Fixed issue where `TableClient`'s and `TableClientAsync`'s `submitTransaction()` and `submitTransactionWithResponse()` could not be called from inside a Reactor thread (e.g. calling it inside a chain of reactive operations, such as `myOtherOperation.then(result -> client.submitTransaction(transactionActions)`).
+- Fixed issue that would make an exception be raised when calling `submitTransaction()` ifJava's SecurityManager was enabled and no `ReflectPermission` had been granted.
```

Comment:
"Fixed issue where an exception would be raised when calling..."
"ifJava" -> "if Java"